packages/ia-userlist-settings/test/ia-user-list-settings.test.ts

```diff
@@ -71,10 +75,19 @@ describe('IAUserListSettings', () => {
 
     const newListElement = el.shadowRoot?.querySelector('.new-list');
     console.log(newListElement);
-    const listId = newListElement?.querySelector('#id');
-    const listName = newListElement?.querySelector('#name');
+    const listId = newListElement?.querySelector('#id') as HTMLInputElement;
```

Comment:
@nsharma123 
I tried to fix these tests. Please take another look at 
`it('fields should have data while edit', async () => {`
There's a problem here. If you can fix it, please commit.
Otherwise I'll look at it in the morning.

Thanks!

Reply:
Run `yarn test`

Reply:
It seems you have already fixed. Thanks!